libs/upstage/README.md

```diff
@@ -23,3 +23,24 @@ See a [usage example](https://python.langchain.com/docs/integrations/text_embedd
 
 Use `solar-embedding-1-large` model for embeddings. Do not add suffixes such as `-query` or `-passage` to the model name.
 `UpstageEmbeddings` will automatically add the suffixes based on the method called.
+
+## LayoutAnalysis Loader
+
+See a [usage example](https://python.langchain.com/v0.1/docs/integrations/document_loaders/upstage/)
+
+To load an image using UpstageLayoutAnalysisLoader, OCR must be enabled by setting `use_ocr=True` when constructing the loader, as shown in the example below:
```

Comment:
Q - are we not going to update the default value of this parameter when input is image?

Reply:
I have added a detailed description of the available OCR options to the README.md file.